src/ext.rs

```diff
@@ -737,7 +738,25 @@ impl TcpStreamExt for TcpStream {
         Ok(Some((secs as u32) * 1000))
     }
 
-    #[cfg(unix)]
+    #[cfg(target_os = "openbsd")]
```

Comment:
Style nit: Would you mind moving these two function definitions down to after their respective counterparts with the `#[cfg(all(unix, not(target_os = "openbsd")))]` gates?  I think it would be nice list the specific `openbsd` case after the more general `unix` case.

Reply:
Sure.  Done in f98618d.